Wiki Update : Add information about Gradle and ktfmt (#50)
epfl-SDP/android: wiki/Home.md

## Pull request

- Each PR must be linked to the corresponding issue.
- You can use the Draft PR, if you want an early review
- For the reviewer, you can prefix your comments with a keyword that indicating the kind of comment
  it is, here is a short list of keywords :
    - Important
    - Nitpick
    - Question
    - Bug
    - Proposition

## Gradle

Make sure to have gradle tasks in the gradle tab inside Android studio.

> ![ExepectedGradle](image/ExpectedGradle.png)

Reviewer: ```suggestion
![ExpectedGradle](image/ExpectedGradle.png)
```

> If you tab isn't like the one on the screen shot, go to `File->Settings...->Experimental`,

Reviewer: ```suggestion
If your tab isn't like the one on the screen shot, go to `File->Settings...->Experimental`,
```

> If your tab isn't like the one on the screen shot, go to `File->Settings...->Experimental`,

Reviewer 2: ```suggestion
If your tab isn't like the one on the screenshot, go to `File->Settings...->Experimental`,
```

> inside `Gradle` section, uncheck `Do not build Gradle task list during Gradle sync`

Reviewer 2: Proposition: This step alone didn't help to show Gradle tasks tab even after I reloading the IDE. I had to re-build the project. Maybe it's worth to mentioning.
